### ISSUE TITLE
Localfiles type unable to remove data from db and image file from local directory

### ISSUE THREAD

**Issue by rinku9900** [keystonejs/keystone#2340]
I have using Localfiles type to upload multiple files, it upload the files successfully and when later I tried to delete the files through the admin UI it did not remove from the data from the table and local directory. I do not get any errors when removing files. Here is the code snippet from the Model.

```
localImages: {
	type: Types.LocalFiles,
	dest: './public/images',
	prefix: '/images',
	allowedTypes: ['image/jpeg', 'image/png'],
	filename: function (item, file) {
		return item.key + '-' + file.originalname;
	},
	format: function (item, file) {
		return '<img src="' + file.href + '" style="max-width:100px" />';
	}
}
```

**Comment by jacobj** [comment]
+1

**Comment by rinku9900** [comment]
Played around with the LocalFilesType.js file in the Keystone and found out these lines in the code causing files not to deleted.

```
value = _.findWhere(values, { '_id': element_id });           //line no. 120
var value = _.findWhere(values, { '_id': element_id });     // line no. 147
var value = _.findWhere(values, { '_id': element_id });    // line no. 174
```

when I changed the following lines to...

```
value = _.findWhere(values, { 'id': element_id });           // line no. 120
var value = _.findWhere(values, { 'id': element_id });     // line no. 147
var value = _.findWhere(values, { 'id': element_id });     // line no. 174
```

everything seems to be working fine. The '_id' causing the files not be deleted from the database and local file system. 

@JedWatson -  can you please update the code and push the changes to download.

Thanks!

**Comment by morenoh149** [comment]
@rinku9900 if you could submit a PR that'd be great

**Comment by rinku9900** [comment]
@morenoh149  what is PR?

**Comment by AlexYegupov** [comment]
@rinku9900, PR is pull request with correspondent fix

**Comment by jacobj** [comment]
I went ahead and submitted a PR with the mentioned fix.

**Comment by rinku9900** [comment]
Thanks @jacobj !

**Comment by mxstbr** [comment]
Fixed in #2347!
